xapian-core/tests/termgentest.cc

```diff
@@ -735,7 +762,21 @@ static bool test_termgen1()
 	    } else if (strncmp(o, "some", 4) == 0) {
 		o += 4;
 		termgen.set_stemming_strategy(termgen.STEM_SOME);
-	    } else if (strncmp(o, "prefix=", 7) == 0) {
+	    } else if (strncmp(o, "stop=en", 7) == 0) {
+               o += 7;
+               array<const char *, 3> x = {"the", "a", "an"};
```

Comment:
@jaylett : Should I change the above line to array<const char *, 3> x = {{"the", "a", "an"}};

I read on this link that Clang rejects list-initialization with single braces([link](http://stackoverflow.com/questions/16985687/brace-elision-in-stdarray-initialization)).